doc/programmatic-api.md

```diff
@@ -143,3 +143,11 @@ of the specified state for specified browser.
 * `gemini.getBrowserCapabilites(browserId)` – returns WebDriver capabilities for specified `browserId`.
 * `gemini.browserIds` – list of all browser identificators to use for tests.
 
+## Events
+
+`gemini` instance emits number of events, which can be used by external scripts or plugins:
+
+* `startRunner` - emitted before the start of test or gather commands. If you return a promise
+from event handler, start of the command will be delayed until promise resolves.
```

Comment:
from the event handler, the start of the command will be delayed until the `promise` resolves.